### PR TITLE
Added preliminary support for Nuvoton NCT6686D chip.

### PR DESCRIPTION
**Pull request by intositeme**
Tested fan controls for
 - ASRock B650E PG-ITX WiFi A discord user test working on
 - ASRock Z690 Phantom Gaming ITX/TB4

Uncertain if voltages are accurate.

### CODE REVIEW

File: LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs [hunk 1]
```diff
@@ -417,7 +418,8 @@ public void Update()
 
         for (int i = 0; i < Voltages.Length; i++)
         {
-            if (Chip != Chip.NCT6687D)
+            // if (Chip != Chip.NCT6687D)
+            if (Chip is not Chip.NCT6687D and Chip.NCT6686D)
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs:422]: You missed a "not" here I think.

Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs:422]: and rebase.

Comment (intositeme) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs:422]: Awesome. I changed it on my files. New to Github, how do i commit it to this?

Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs]: With GitHub Desktop you can rebase (Branch menu item, somewhere at the bottom).

Comment (intositeme) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs]: Awesome. I think i did it. :)

Comment (intositeme) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs]: image
Seeing more voltage items.
Would need help here, as I'm unsure about them.

Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs]: These values look odd, but I'm not sure this chipset is nct6686d, it should have id 0xd440.

Comment (intositeme) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs]: I got the ID `0xD441` off HWinfo's export.

Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs]: I'm not sure that's correctly identified by hwinfo then, it should be 0xd440: https://github.com/torvalds/linux/blob/master/drivers/hwmon/nct6683.c#L72
Which might explain the odd values you're seeing.

Comment (intositeme) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs]: Just tried 0xd440, but the first 9 values are still not showing.

File: LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs [hunk 2]
```diff
@@ -872,7 +874,7 @@ private void RestoreDefaultFanControl(int index)
     {
         if (_restoreDefaultFanControlRequired[index])
         {
-            if (Chip is not Chip.NCT6687D and not Chip.NCT6683D)
+            if (Chip is not Chip.NCT6687D and not Chip.NCT6686D)
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs:877]: Restore ```and not Chip.NCT6683D```.

File: LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/LpcIO.cs [hunk 3]
```diff
@@ -377,6 +377,10 @@ private bool DetectWinbondFintek(LpcPort port)
                         chip = Chip.NCT6798D;
                         logicalDeviceNumber = WINBOND_NUVOTON_HARDWARE_MONITOR_LDN;
                         break;
+                    case 0x41:
```
[the remark this answers is not in the document]
Comment (intositeme) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/LpcIO.cs:380]: I changed this to 0x40, and all the displays does not show up. 
image

[the remark this answers is not in the document]
Comment (intositeme) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/LpcIO.cs:380]: @PhyxionNL as mentioned, tried changing all to 0x40, the motherboard doesn't even show any info

[the remark this answers is not in the document]
Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/LpcIO.cs:380]: That seems likely as you seem to have another chipset. The changes here are mostly correct (see notes) for the NCT6686D chip. What you do have though, I'm not sure. I cannot find it. Perhaps you can check your motherboard physically and see what's written on it.

[the remark this answers is not in the document]
Comment (secretformula) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/LpcIO.cs:380]: Following up here, I tried running this nightly and also do not see displays show up with my B650E PG-ITX. Will try charging this back to 0x41

[the remark this answers is not in the document]
Comment (mrsideshowjack) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/LpcIO.cs:380]: > Following up here, I tried running this nightly and also do not see displays show up with my B650E PG-ITX. Will try charging this back to 0x41
Same for my `ASRock Z590 Phantom Gaming ITX/TB4`

[the remark this answers is not in the document]
Comment (secretformula) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/LpcIO.cs:380]: I can confirm what @intositeme saw, only seeing information display on my B650E PG-ITX WIFI (Windows 11) if I change this constant back to 0x41. Created a new ticket #911 for resolving. Much appreciation for this initial support / reverse engineering.

**Pull request closed without merging**